DataFormats/ForwardDetId/interface/HGCTriggerDetId.h

```diff
@@ -0,0 +1,96 @@
+#ifndef DataFormats_ForwardDetId_HGCTriggerDetId_H
+#define DataFormats_ForwardDetId_HGCTriggerDetId_H 1
+
+#include <iosfwd>
+#include "DataFormats/DetId/interface/DetId.h"
+#include "DataFormats/ForwardDetId/interface/ForwardSubdetector.h"
+#include "TObject.h"
+
+
+class HGCTriggerDetId : public DetId {
+  // |   DetId           | HGCTriggerDetId 
+  // | 1111     | 111    | 1     | 11111 | 1      | 111111 | 111111111111
+  // | detector | subdet | zside | layer | subsec | sector | cell
+  // | 15       | 7      | 2     | 31    | 2      |  63    | 4096   
+  // const static uint32_t cell_shift = 0;
+  // const static uint32_t cell_mask = 0xFFF;
+  // const static uint32_t sector_shift = 12;
+  // const static uint32_t sector_mask = 0x3F;
+  // const static uint32_t subsector_shift = 18;
+  // const static uint32_t subsector_mask = 0x1;
+  // const static uint32_t layer_shift = 19;
+  // const static uint32_t layer_mask = 0x1F;
+  // const static uint32_t zside_shift = 24;
+  // const static uint32_t zside_mask = 0x1;
+  
+  // |   DetId           | HGCTriggerDetId 
+  // | 1111     | 111    | 1     | 11111 | 1111111 | 111111 | 111111
+  // | detector | subdet | zside | layer | sector  | mod    | cell
+  // | 15       | 7      | 2     | 31    |  127    | 64     | 64
+
+  const static uint32_t cell_shift = 0;
+  const static uint32_t cell_mask = 0x3F;
```

Comment:
@amarini 6 bits for the cell field is not enough. There is no room left for UndefinedCell, all available values (including 0x3F) are used by actual cells in the current configuration. Also it would be good to have some margin. So I would use 1 bit more here, taken from the sector field (64 sectors is enough).